wordpress-mobile/gutenberg-mobile
# Fix missing Featured Image translations

Fix missing translations on Android
See GB : https://github.com/WordPress/gutenberg/pull/37956

Partial Fix for #4432

To test:
See test step on https://github.com/WordPress/gutenberg/pull/37956

**NOTE:** This issue only affects the Android platform.
_(Edited by @fluiddot)_

PR submission checklist:
